wagtailio/blog/templates/blog/includes/blog_post.html

```diff
@@ -0,0 +1,29 @@
+{% load wagtailimages_tags %}
+<a href="{{ post.url }}" class="blog-post {% if featured %} blog-post--featured{% else %} blog-post--small{% endif %}">
+    {% if featured %}
+        <div class="blog-post__image-wrapper" aria-hidden="true">
+            {% if post.listing_image %}
+                {% image post.listing_image fill-500x500 class='blog-post__image' %}
+            {% elif post.main_image %}
+                {% image post.main_image fill-500x500 class='blog-post__image' %}
+            {% else %}
+                {# TODO: Render a default image #}
```

Comment:
I think we should expect editors to add images to featured posts but let's leave this in to add something if we have time.